core/src/main/java/bio/terra/pearl/core/service/datarepo/DataRepoClient.java

```diff
@@ -60,6 +63,7 @@ public JobModel ingestDataset(UUID spendProfileId, UUID datasetId, String tableN
                 .table(tableName)
                 .profileId(spendProfileId)
                 .format(IngestRequestModel.FormatEnum.CSV)
+                .csvFieldDelimiter("\t")
```

Comment:
🤔 CSV is literally "comma-separated values", so I'm not sure what it means to use tab as the field delimiter. 🤷‍♂️

Reply:
Hah yeah, I was a bit surprised by this as well. I think we'll support CSV export in Juniper at some point so that might be the cleaner way of handling this but for now it seems to work well.